tests/components/nina/test_binary_sensor.py

```diff
@@ -0,0 +1,253 @@
+        assert state_w1.attributes.get(ATTR_EXPIRES) == ""
+
+        assert entry_w1.unique_id == "083350000000-1"
+        assert entry_w1.device_class == DEVICE_CLASS_SAFETY
+
+        state_w2 = hass.states.get("binary_sensor.warnung_aach_stadt_2")
+        entry_w2 = entity_registry.async_get("binary_sensor.warnung_aach_stadt_2")
+
+        assert state_w2.state == STATE_ON
+        assert state_w2.attributes.get(ATTR_HEADLINE) == "Ausfall Notruf 112"
+        assert state_w2.attributes.get(ATTR_ID) == "mow.DE-NW-BN-SE030-20201014-30-000"
+        assert state_w2.attributes.get(ATTR_SENT) == "2021-10-11T05:20:00+01:00"
+        assert state_w2.attributes.get(ATTR_START) == "2021-11-01T05:20:00+01:00"
+        assert state_w2.attributes.get(ATTR_EXPIRES) == "3021-11-22T05:19:00+01:00"
+
+        assert entry_w2.unique_id == "083350000000-2"
+
+        state_w3 = hass.states.get("binary_sensor.warnung_aach_stadt_3")
+        entry_w3 = entity_registry.async_get("binary_sensor.warnung_aach_stadt_3")
+
+        assert state_w3.state == STATE_OFF
+        assert state_w3.attributes.get(ATTR_HEADLINE) == ""
+        assert state_w3.attributes.get(ATTR_ID) == ""
+        assert state_w3.attributes.get(ATTR_SENT) == ""
+        assert state_w3.attributes.get(ATTR_START) == ""
+        assert state_w3.attributes.get(ATTR_EXPIRES) == ""
+
+        assert entry_w3.unique_id == "083350000000-3"
+        assert entry_w3.device_class == DEVICE_CLASS_SAFETY
+
+        state_w4 = hass.states.get("binary_sensor.warnung_aach_stadt_4")
+        entry_w4 = entity_registry.async_get("binary_sensor.warnung_aach_stadt_4")
+
+        assert state_w4.state == STATE_OFF
+        assert state_w4.attributes.get(ATTR_HEADLINE) == ""
+        assert state_w4.attributes.get(ATTR_ID) == ""
+        assert state_w4.attributes.get(ATTR_SENT) == ""
+        assert state_w4.attributes.get(ATTR_START) == ""
+        assert state_w4.attributes.get(ATTR_EXPIRES) == ""
+
+        assert entry_w4.unique_id == "083350000000-4"
+        assert entry_w4.device_class == DEVICE_CLASS_SAFETY
+
+        state_w5 = hass.states.get("binary_sensor.warnung_aach_stadt_5")
+        entry_w5 = entity_registry.async_get("binary_sensor.warnung_aach_stadt_5")
+
+        assert state_w5.state == STATE_OFF
+        assert state_w5.attributes.get(ATTR_HEADLINE) == ""
+        assert state_w5.attributes.get(ATTR_ID) == ""
+        assert state_w5.attributes.get(ATTR_SENT) == ""
+        assert state_w5.attributes.get(ATTR_START) == ""
+        assert state_w5.attributes.get(ATTR_EXPIRES) == ""
+
+        assert entry_w5.unique_id == "083350000000-5"
+        assert entry_w5.device_class == DEVICE_CLASS_SAFETY
+
+
+async def test_sensors_without_corona_filter(hass: HomeAssistant) -> None:
+    """Test the creation and values of the NINA sensors without the corona filter."""
+
+    dummy_response: Dict[str, Any] = json.loads(
+        load_fixture("nina/sample_warnings.json")
```

Comment:
```suggestion
        load_fixture("sample_warnings.json", "nina")
```

This was recently refactored. The old way is only there for backwards compatibility. Please change your fixture path to the components tests folder.